src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurCashAdvancePdpFeedFileServiceImpl.java

```diff
@@ -51,32 +53,73 @@ public boolean createPdpFeedFileForValidatedDetailFileLines(ConcurRequestExtract
         return pdpFileSuccessfullyCreated;
     }
 
-    private PdpFeedFileBaseEntry buildPdpFeedBaseEntry(ConcurRequestExtractFile requestExtractFile) {
+    private PdpFeedFileBaseEntry buildPdpFeedBaseEntry(ConcurRequestExtractFile requestExtractFile, ConcurRequestExtractBatchReportData reportData) {
```

Comment:
The "longness" is due to the addition of the processing needed for the report data into a function that is mainly control flow. See my comment below.